MailCore/Utils/Constants.swift

```diff
@@ -161,8 +161,16 @@ public enum Constants {
         let release = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String? ?? "x.x"
         let build = Bundle.main.object(forInfoDictionaryKey: "CFBundleVersion") as! String? ?? "x"
         let betaRelease = Bundle.main.isRunningInTestFlight ? "beta" : ""
+        let platformDetector = PlatformDetector()
+        var device = ""
 
-        return "\(appName) iOS version \(release)-\(betaRelease)\(build)"
+        if platformDetector.isMac {
```

Comment:
Can you check if we can get the name with this property?
https://developer.apple.com/documentation/uikit/uidevice/1620054-systemname